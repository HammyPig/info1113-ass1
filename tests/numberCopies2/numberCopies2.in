ADD MEMBER coolman69
nUMBER copies
ADD COLLECTION duplicates.csv
NUMBER copIES
RENT 100000 111111
number cOPIES
RENT 100000 111112
RENT 100000 111113
number COPIES
RELINQUISH ALL 100000
NUMBER COPIES