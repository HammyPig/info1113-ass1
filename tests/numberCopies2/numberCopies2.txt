accurate number of copies (including duplicates and complex interaction) in alphabetical order