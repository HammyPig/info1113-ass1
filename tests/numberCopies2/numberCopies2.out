user: Success.

user: No books in system.

user: 3 books successfully added.

user: The Iliad (Homer): 1
To Kill a Mockingbird (Harper Lee): 2

user: Success.

user: The Iliad (Homer): 1
To Kill a Mockingbird (Harper Lee): 2

user: Success.

user: Success.

user: The Iliad (Homer): 1
To Kill a Mockingbird (Harper Lee): 2

user: Success.

user: The Iliad (Homer): 1
To Kill a Mockingbird (Harper Lee): 2

user: Ending Library process.
